# Glass gaze
Header header
float32 u_2d
float32 v_2d
int32 width
int32 height